transport/dtls/SrtpClient.cpp

```diff
@@ -306,6 +306,11 @@ bool SrtpClient::unprotect(memory::Packet& packet)
             return false;
         }
     }
+    else
+    {
+        return false;
```

Comment:
I guess the only reason to code hit this line is due a bug, right? As we are explicitly handle this case it wouldn't be better write na error log to not be blind about this when it happens?

Reply:
or perhaps assert(false). We will have checked already that it is an rtp packet. 
But there is also a problem if client sneds us real rtp header but rest of packet cannot be unprotected due to it is corrupt. It will cause log spam.